shared/elements/gaia_grid_rs/style.css

```diff
@@ -409,3 +409,35 @@ instead of the context menu.
 :-moz-dir(rtl) gaia-grid-rs {
   direction: ltr;
 }
+
+
+gaia-grid-rs {
+  display: block;
+  height: 20rem !important;
```

Comment:
Yes, we need. 
It is a temporary solution.
When we will start to clean gaia_grid_rs module, we will fix it